src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -535,6 +534,55 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(e, e.Items);
                 }
             }
+
+            if (e.BuildEventContext == null || e.Items == null)
+            {
+                return;
+            }
+            // node and project context ids for the propertyOutputMap key
+            int nodeID = -1;
+            int projectContextId = -1;
+            nodeID = e.BuildEventContext.NodeId;
+            projectContextId = e.BuildEventContext.ProjectContextId;
+
+            // Create the value to be added to the propertyOutputMap
+            StringBuilder logOutputProperties = new StringBuilder();
+
+            foreach (DictionaryEntry item in e.Items)
+            {
+                ITaskItem itemVal = (ITaskItem)item.Value;
+                // Determine if the LogOutputProperties item has been used
+                if (string.Equals((string)item.Key, "LogOutputProperties", StringComparison.OrdinalIgnoreCase))
+                {
+                    // Look for the property value associated with the property key
+                    // Note: the property key is the item value
+                    string value = null;
+                    bool foundProperty = e.GlobalProperties.TryGetValue(itemVal.ItemSpec, out value);
+     
+                    // Look for the property in local properties if it wasn't found in global properties
```

Comment:
Do we want this? Wouldn't it then apply to single-targeted SDK projects, where it doesn't really make sense?